cmd/wstail/tail.go

```diff
@@ -24,23 +25,24 @@ import (
 
 const defaultTailSource = "/var/log/messages"
 
-// tailHistory defines the number of lines to output
-const tailHistory = 100
+// maxDumpLen defines the maximum size of the string window to output
+// to console after a failed interpretation (as in failure to decode JSON)
+const maxDumpLen = 128
 
 func tailer(ws *websocket.Conn, filter filter) {
 	matcher := buildMatcher(filter)
 	log.Infof("active filter: %v (%v)", filter, matcher)
 	defer ws.Close()
 
-	var err error
-	var history io.ReadCloser
-	tailCmd := exec.Command("tail", "-f", filePath)
+	rotated, err := collectRotatedMessages()
+	if err != nil {
+		log.Errorf("failed to read a list of rotated log files: %v", err)
```

Comment:
it is ok as far as not showing logs from rotated files goes.
I'll add a test for the function so I can rule any logic errors.